webpack.config.js

```diff
@@ -234,22 +232,6 @@ module.exports = function (_, env) {
         analyzerMode: 'static',
         defaultSizes: 'gzip',
         openAnalyzer: false
-      }),
-
-      // Generate a ServiceWorker using Workbox.
-      isProd && new WorkboxPlugin.GenerateSW({
-        swDest: 'sw.js',
-        clientsClaim: true,
-        skipWaiting: true,
-        exclude: [
-          'report.html',
-          'manifest.json',
-          /(report\.html|manifest\.json|\.precache-manifest\..*\.json)$/,
-          /\.(?:map|pem|DS_Store)$/
-        ],
-        // allow for offline client-side routing:
-        navigateFallback: '/',
-        navigateFallbackBlacklist: [/\.[a-z0-9]+$/i]
       })
```

Comment:
I don't think it's time to add a service worker yet. Also, they shouldn't be prod only, else it becomes easy to forget and break the site.

Reply:
Any harm? I did it because we know this will be a PWA, and our requirements are pretty basic - static files with deterministic hashed names, all precached aggressively.

Reply:
> static files with deterministic hashed names, all precached aggressively

tbqh that’s not basic in my book. 

It will be interesting for a lot of people to document how we went from a non-PWA preact app to a full PWA one.